azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License.
 ##
 from collections import defaultdict
+import numpy as np

```

Comment:
Are we adding `numpy` as a dependency just for this? Since a bit heavy just to generate a random number.